app/assets/stylesheets/application.css.scss

```diff
@@ -91,6 +91,28 @@ input {
   padding-top: 12%;
 }
 
+.jumbotron > #form-c {
+  background: none !important;
+  text-align: center;
+}
+
+.jumbotron > form > input[type="text"] {
+ border-radius: 4px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered border, border-radius, height, margin-top, text-align, width